modules/nw-ingress-sharding.adoc

```diff
@@ -1,17 +1,105 @@
+
+This lets you distribute subsets of routes over multiple Ingress Controllers. These subsets can be non-overlapping, also called _traditional_ sharding, or overlapping, otherwise known as _overlapped_ sharding.
+
+== Traditional sharding example
+
+An Ingress Controller `finops-router` is configured with the label selector `spec.namespaceSelector.matchLabels.name` set to `finance` and `ops`:
+
+.Example YAML definition for `finops-router`
+[source,yaml]
+----
+apiVersion: v1
+items:
+- apiVersion: operator.openshift.io/v1
+  kind: IngressController
+  metadata:
+    name: finops-router
+    namespace: openshift-ingress-operator
+  spec:
+    namespaceSelector:
+      matchLabels:
+        name:
+          - finance
+          - ops
+----
+
+A second Ingress Controller `dev-router` is configured with the label selector `spec.namespaceSelector.matchLabels.name` set to `dev`:
+
+.Example YAML definition for `dev-router`
+[source,yaml]
+----
+apiVersion: v1
+items:
+- apiVersion: operator.openshift.io/v1
+  kind: IngressController
+  metadata:
+    name: dev-router
+    namespace: openshift-ingress-operator
+  spec:
+    namespaceSelector:
+      matchLabels:
+        name: dev
+----
+
+If all application routes are in separate namespaces, each labeled with `name:finance`, `name:ops`, and `name:dev` respectively, this configuration effectively distributes your routes between the two Ingress Controllers. {product-title} routes for console, authentication, and other purposes should not be handled.
+
+In the above scenario, sharding becomes a special case of partitioning, with no overlapping subsets. Routes are divided between router shards.
+
+[WARNING]
+====
+The `default` Ingress Controller will continue to serve all routes unless the `namespaceSelector` or `routeSelector` fields contain routes that are meant for exclusion. See this link:https://access.redhat.com/solutions/5097511[Red Hat Knowledgebase solution] and the section "Sharding the default Ingress Controller" for more information on how to exclude routes from the default Ingress Controller.
+====
+
+== Overlapped sharding example
+
+In addition to `finops-router` and `dev-router` in the example above, you also have `devops-router`, that is configured with the label selector `spec.namespaceSelector.matchLabels.name` set to `dev` and `ops`:
+
+.Example YAML definition for `devops-router`
+[source,yaml]
+----
+apiVersion: v1
+items:
+- apiVersion: operator.openshift.io/v1
+  kind: IngressController
+  metadata:
+    name: devops-router
+    namespace: openshift-ingress-operator
+  spec:
+    namespaceSelector:
+      matchLabels:
+        name:
+          - dev
+          - ops
+----
+The routes in the namespaces labeled `name:dev` and `name:ops` are now serviced by two different Ingress Controllers. This becomes a case in which you have defined overlapping subsets of routes.
```

Comment:
"This becomes a case in which you have defined" -> "With this configuration, you have" 

Assuming I have the idea right.